src/main/java/org/gridsuite/sensitivityanalysis/server/service/SensitivityAnalysisService.java

```diff
@@ -146,4 +158,62 @@ private Integer getContingenciesCount(List<UUID> ids, UUID networkUuid, String v
     public String getDefaultProvider() {
         return defaultProvider;
     }
+
+    public byte[] exportSensitivityResultsAsCsv(UUID resultUuid, SensitivityAnalysisCsvFileInfos sensitivityAnalysisCsvFileInfos) {
+        if (sensitivityAnalysisCsvFileInfos == null ||
+                sensitivityAnalysisCsvFileInfos.getSensitivityFunctionType() == null ||
+                sensitivityAnalysisCsvFileInfos.getResultTab() == null ||
+                CollectionUtils.isEmpty(sensitivityAnalysisCsvFileInfos.getCsvHeaders())) {
+            throw new PowsyblException("Missing information to export sensitivity result as csv : Sensitivity result tab, sensitivity function type and csv file headers must be provided");
```

Comment:
In my opinon, we need create a handler RestResponseEntityExceptionHandler and SensibilityAnalysisException as in study-server and security-analysis-server => pass a type INVALID_PARAMS  and a message. In the handler, return response entity with message

Reply:
Done